### PR TITLE
advanced config update

### PR DESCRIPTION
**Pull request by rosieyohannan**
Updating the Advanced config section to highlight orbs, fix some errors and improve interlinking from the overview page.

How it looks now:
Screenshot 2020-03-04 at 12 57 44

With links to relevant sections:
Screenshot 2020-03-04 at 12 58 10

### CODE REVIEW

File: jekyll/_cci2/browser-testing.md [hunk 1]
```diff
@@ -173,78 +175,20 @@ build:
 
 ### LambdaTest Browser Testing Orb Example
 
-LambdaTest has developed a CircleCI orb for browser compatibility testing that enables you to open a Lambda Tunnel before performing any browser testing. Also, to ease the process of integration between LambdaTest & CircleCI. This orb (a package of configurations that you can use in your workflow) has been developed and certified for use and can simplify your configuration workflows. An example of the orb is shown below.
+LambdaTest has developed a [CircleCI orb](https://circleci.com/orbs/registry/orb/lambdatest/lambda-tunnel) for browser compatibility testing that enables you to open a Lambda Tunnel before performing any browser testing, easing the process of integrating LambdaTest with CircleCI. Use the orb to quickly set up a Lambda tunnel and the define your test steps
+
 {% raw %}
-```
```
Comment (rosieyohannan) [inline jekyll/_cci2/browser-testing.md]: This was just a straight copy of the orb source. replacing with a usage example...

File: jekyll/_cci2/building-docker-images.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: classic-docs
-title: "Running Docker Commands"
-short-title: "Running Docker Commands"
+title: "Building Docker Images"
```
Comment (rosieyohannan) [inline jekyll/_cci2/building-docker-images.md]: Happy to switch this back depending on what others think. Building Docker Images just makes more sense to me as a title.

File: jekyll/_cci2/pipeline-variables.md [hunk 3]
```diff
@@ -194,10 +194,7 @@ jobs:
   mytestjob:
     steps:
       - checkout
-      - when:
```
Comment (rosieyohannan) [inline jekyll/_cci2/pipeline-variables.md]: This usage of `when` is not supported.

**Comment by rosieyohannan** [comment]
@teesloane hey! if you have time to review my PR, would you also have a quick look and see if you can work out why the build is failing on the code snippets? I've looked at it for a while but I can't see why...

File: jekyll/_cci2/building-docker-images.md (hunk 2 above)
Comment (teesloane) [reply jekyll/_cci2/building-docker-images.md]: I believe there are other docker commands a user might be using, though (although most prevalently ... `build`-ing...) I'm not super versed on Docker stuff, though.

Comment (rosieyohannan) [reply jekyll/_cci2/building-docker-images.md]: Oh yeah! Good point 👍

File: jekyll/_cci2/browser-testing.md [hunk 4]
```diff
@@ -54,197 +54,28 @@ For more information about working with Headless Chrome,
...
-      tunnel_name:
-        description: Name your lambda tunnel and make sure to use this tunnel_name desired capability in your test
-        type: string
-        default: ""
-      steps:
-        type: steps
-        description: Steps to execute once the Lambdatest Tunnel is available
-    docker: 
-      - image: <<parameters.docker>>
+  lambdatest/with_tunnel
```
Comment (teesloane) [inline jekyll/_cci2/browser-testing.md]:
```suggestion
  lambdatest/with_tunnel:
```
I think this might be what's causing the failing build.

Comment (rosieyohannan) [reply jekyll/_cci2/browser-testing.md]: Oh well, worth a try, thanks Tyler! :-)

File: jekyll/_cci2/building-docker-images.md (hunk 2 above)
Comment (rosieyohannan) [inline jekyll/_cci2/building-docker-images.md]:
```suggestion
title: "Running Docker Commands"
```

File: jekyll/_cci2/building-docker-images.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: classic-docs
 title: "Running Docker Commands"
-short-title: "Running Docker Commands"
+short-title: "Building Docker Images"
```
Comment (rosieyohannan) [inline jekyll/_cci2/building-docker-images.md]:
```suggestion
short-title: "Running Docker Commands"
```

**Pull request merged**